packages/sdk/src/contracts/holograph.contract.ts

```diff
@@ -83,7 +83,7 @@ export class Holograph {
 
   /**
    * Get the address of the Holograph Bridge module.
-   * Used for beaming holographable assets cross-chain.
+   * Used for bridging holographable assets cross-chain.
```

Comment:
haha finally. wish we never used "beaming". too bad we can't easily make the same changes in the protocol repo 🥲